gcloud/streaming/transfer.py

```diff
@@ -1,37 +1,54 @@
 # pylint: skip-file
-"""Upload and download support for apitools."""
-from __future__ import print_function
+"""Upload and download support for apitools.
+
+:mod:`gcloud.storage.blob` uses:
+
+- :class:`Download`
+- :class:`Upload`
+- :const:`RESUMABLE_UPLOAD`
+
+:mod:`gcloud.storage.test_blob` patches:
```

Comment:
6891504 drops them altogether:  they are notes-to-myself to make it easier to remove features we don't use.